_pages/about/code-of-conduct.md

```diff
@@ -6,3 +6,7 @@
 ### Code of Conduct
 
 All members of the club are expected to follow the policies and ideals detailed in [Hand and Mind](https://handbook.mit.edu/).
+
+MITOC maintains a “zero-tolerance” policy (resulting in removal from the club) for the following behaviors: stalking, violent threats of any kind, physical violence (including sexual assault), domestic/dating violence, and sexual harassment. Reach out to [mitoc-mediation@mit.edu](mitoc-mediation@mit.edu) with questions/concerns.
```

Comment:
Only on my phone so it’s a bit tricky to make a code suggestion but you need a “mailto:” on the email link!